src/styles/blocks/content.scss

```diff
@@ -0,0 +1,6 @@
+.content {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  height: 100vh;
```

Comment:
better is to use min-height, instead of height bcs your content can become bigger than 100vh, so its expands,
but that is not a problem with the task so Good job.

Reply:
In general you're right, but it won't happen in this case - he took care of it 🥇